src/support/scale.rs

```diff
@@ -75,6 +75,19 @@ impl<const BITS: usize, const LIMBS: usize> CompactAs for CompactUint<BITS, LIMB
     }
 }
 
+impl<const BITS: usize, const LIMBS: usize> MaxEncodedLen for CompactUint<BITS, LIMBS> {
+    fn max_encoded_len() -> usize {
+        Uint::<BITS, LIMBS>::max_encoded_len()
+    }
+}
+
+impl<const BITS: usize, const LIMBS: usize> Encode for CompactUint<BITS, LIMBS> {
+    fn encode_to<T: Output + ?Sized>(&self, dest: &mut T) {
+        let v: CompactRefUint<BITS, LIMBS> = (&self.0).into();
+        v.encode_to(dest)
+    }
+}
```

Comment:
I need to implement this as MaxEncodedLen inherits from Encode, we can't get away without it